running migrations from Makefile not working

### reproducing the error

run these commands in order:<br/>
- `make postgres`<br/>
result:<br/>
```
docker rm -f postgres-rmx &> /dev/null || true
docker run --name postgres-rmx -e POSTGRES_PASSWORD=postgrespw -p 9000:9000 -e POSTGRES_USER=rmx -d postgres:14.6-alpine
f370b4c3d7093e26eb795c9280fed299b447656ad4fa32b2b15010f82144aa97
```
- `make createdb`<br/>
result:<br/>
```
docker exec -it  postgres-rmx createdb --username=rmx --owner=rmx rmx-dev
```
- `make migrateup`<br/>
result:<br/>
```
migrate -path internal/db/migration -database "postgresql://rmx:postgrespw@localhost:5432/rmx-dev?sslmode=disable" -verbose up
2023/01/15 17:26:18 error: EOF
make: *** [Makefile:68: migrateup] Error 1
```
<br/>

--- a/Makefile
+++ b/Makefile
@@ -72,11 +72,11 @@ dropdb:
 
 .PHONY: migrateup
 migrateup:
-	migrate -path internal/db/migration -database $(PG_CONN_STRING) -verbose up
+	migrate -path internal/jam/postgres/migration -database $(PG_CONN_STRING) -verbose up
 
 .PHONY: migratedown
 migratedown:
-	migrate -path internal/db/migration -database $(PG_CONN_STRING) -verbose down
+	migrate -path internal/jam/postgres/migration -database $(PG_CONN_STRING) -verbose down
 
 .PHONY: sqlc
 sqlc:

